support-workers/src/test/scala/com/gu/zuora/ZuoraITSpec.scala

```diff
@@ -0,0 +1,147 @@
+package com.gu.zuora
```

Comment:
I even used the `git mv` function to try to keep the history but it didn't work!  This is a rename of ZuoraSpec, as I wanted an actual ZuoraSpec that wasn't an IT one.